Makefile

```diff
@@ -35,24 +43,24 @@ man:
 runctestimage:
 	docker build -t $(RUNC_TEST_IMAGE) -f $(TEST_DOCKERFILE) .
 
-test: runctestimage
+test:
+	make unittest integration
+
+localtest:
+	make localunittest localintegration
+
+unittest: runctestimage
 	docker run -e TESTFLAGS -ti --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_TEST_IMAGE) make localtest
 
-localtest: all
+localunittest: all
 	go test -tags "$(BUILDTAGS)" ${TESTFLAGS} -v ./...
 
-dbuild: runctestimage
-	docker build -t $(RUNC_IMAGE) .
-	docker create --name=$(RUNC_INSTANCE) $(RUNC_IMAGE)
-	docker cp $(RUNC_INSTANCE):$(RUNC_BUILD_PATH) .
-	docker rm $(RUNC_INSTANCE)
-
 integration: runctestimage
 	docker run -e TESTFLAGS -t --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_TEST_IMAGE) make localintegration
 
-localintegration:
+localintegration: all
 	bats tests/integration${TESTFLAGS}
```

Comment:
Would it be possible to always run this in TAP mode via `-t` while at it? Jenkins doesn't log very nicely the output otherwise.